### PR TITLE
QE lensing reconstruction

### PR DESCRIPTION
**Pull request by louisl3grand**
This contains the code that I used to do the lensing QE for the chwide component separated maps. There is a README explaining the details of the reconstruction, and also a notebook and the slides I presented at the last meeting.

### CODE REVIEW

File: 20240531_reconstructed_cmblens/README.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Reconstucted CMB lensing fields
+
+Louis Legrand (louis.legrand@outlook.com)
+
+The lensing fields are reconstructed from the simulated chwide component separated maps, stored in `/global/cfs/cdirs/cmbs4xlb/v1/component_separated/chwide`
```
Comment (zonca) [inline 20240531_reconstructed_cmblens/README.md]: I might have missed this, do you have some docs about these component  separated maps? can you add it here?

**Review by zonca — APPROVED** [review]
very good, thanks!
just a minor comment

File: 20240531_reconstructed_cmblens/README.md (hunk 1 above)
Comment (louisl3grand) [reply 20240531_reconstructed_cmblens/README.md]: These maps were provided by Shamik, but I think there is no docs on this repo yet. Maybe the best would be to ask him to upload it here?

Comment (zonca) [reply 20240531_reconstructed_cmblens/README.md]: @1cosmologist could you create another folder on this repo with docs about those maps?

Comment (zonca) [reply 20240531_reconstructed_cmblens/README.md]: @1cosmologist could you create another folder on this repo with docs about those maps?

Comment (1cosmologist) [reply 20240531_reconstructed_cmblens/README.md]: @zonca Created PR https://github.com/litebird/litebirdXS4/pull/18 to add the documentation for the NILC maps.

**Review by zonca — APPROVED** [review]

**Pull request merged**